cmd/service.go

```diff
@@ -347,13 +347,16 @@ func runServiceStatusCmd(cmd *cobra.Command, args []string) {
 	if viper.GetBool(constants.ArgAll) {
 		showAllStatus()
 	} else {
-		dbInfo, dbInfoErr := db_local.GetStatus()
-		pmInfo, pmInfoErr := plugin_manager.LoadPluginManagerState()
-		if dbInfoErr != nil || pmInfoErr != nil {
-			utils.ShowError(fmt.Errorf("could not get Steampipe service status"))
+		dbState, dbStateErr := db_local.GetState()
+		pmState, pmStateErr := plugin_manager.LoadPluginManagerState()
+
+		if dbStateErr != nil || pmStateErr != nil {
+			utils.ShowError(fmt.Errorf(`could not get Steampipe service status:
```

Comment:
I think a function to construct the error, taking 2 child errors either of which may be nil
If both are nil the function returns nil
Otherwise the we only show the message appropriate to the error